src/meshes/t8code_mesh.jl

```diff
@@ -982,12 +1144,14 @@ function fill_mesh_info!(mesh::T8codeMesh, interfaces, mortars, boundaries,
 
     # Helper variables to compute unique global MPI interface/mortar ids.
     max_level = t8_forest_get_maxlevel(mesh.forest) #UInt64
-    max_tree_num_elements = UInt64(2^ndims(mesh))^max_level
+    max_tree_num_elements = (UInt128(2)^ndims(mesh))^max_level
```

Comment:
Was there a specific reason for converting `2` instead of coverting `2^ndims` as done above?

Reply:
None in particular. I think the result is the same. Maybe in the future we need support for thousands of dimensions ...